docs/contributor_guide/code_style.rst

```diff
@@ -0,0 +1,61 @@
+Code style
+==========
+
+Github conventions
+------------------
+
+Because the Fairlearn team squash merges pull requests, you do not need to put
+much effort into the commit messages you submit when pushing code.
+
+Titles should be descriptive and include one of the following prefixes:
+* DOC: any documentation-related PRs (including the user guide, API reference, and other 
+website-related PRs).
+* MNT: code maintenance (refactoring, improve efficiency, etc.).
+* CI: anything related to our automated tests (nightly builds, CircleCI, releases, etc.).
+* FIX: bug fixes.
+* FEAT/ENH: adds or removes a new feature in the codebase.
+
+Test coverage is checked with ``codecov`` and line missing tests will show up in CI 
+as a failure. Therefore, we recommend contributors ensure all new content 
+they introduce has adequate test coverage.
+
+
+Code conventions
+----------------
+
+Linting
+^^^^^^^
+
+We recommend using a linter to check your code before you submit a PR. 
+We use ``ruff`` to check for PEP8 compatibility issues. You can either follow
+the guidelines, or you can run ``black`` on your code. The generated
+formatting by ``black`` is compatible our formatting requirements. You can
+configure your IDE to use ``black`` to format your code. Please refer to your
+IDE's instructions for further details.
+
```

Comment:
I'd definitely add something about the pre-commit hooks. I don't know if we'd like to reference the other document in another way here except link directly 🤔 

```suggestion
Attaining the project compatible linting is also possible without changing your local setup. You can enable the pre-commit hooks, which will run the linters described above with the settings defined in the [pyproject.toml](https://github.com/fairlearn/fairlearn/blob/main/pyproject.toml). The installation instructions are described in step 6 of `"Contributing a Pull Request"`<https://fairlearn.org/main/contributor_guide/development_process.html#contributing-a-pull-request>`_ .

```

Reply:
Thank you! I wasn't sure what to add about the pre-commit hooks, so I appreciate your input.